### PR TITLE
AX-452-frontend-get-full-athlete-position-names

### PR DESCRIPTION
**Pull request by Friendlygeckos**
# Description
Currently the athlete positions for the scout and athlete page are all set to "B". This ticket replaces the text with the full athlete position names.

Fixes Jira Ticket #452
https://athletex.atlassian.net/jira/software/projects/AX/boards/1?selectedIssue=AX-452

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Is this a UI Change? If so please include screenshot of before and after states below
## Before
Capture
Capture

## After
Capture
Capture

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have assigned 2 reviewers to check my work
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: lib/util/AbbreviationMappingsHelper.dart [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Map<String, String> position = {'RF' : 'Right Fielder', 'SS' : 'Short Stop', '1B' : 'First Baseman', '3B' : 'Third Baseman', 'CF' : 'Center Fielder', 'LF' : 'Left Fielder'};
```
Comment (ketonkss4) [inline lib/util/AbbreviationMappingsHelper.dart:1]: make this a more descriptive title to what type of map this is and maybe consider making this private if you would rather people use the function to get the full name

File: lib/util/AbbreviationMappingsHelper.dart [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+String getFullAthletePosition(String currentAthletePositon) {
```
Comment (ketonkss4) [inline lib/util/AbbreviationMappingsHelper.dart:3]: rename to something that shows that it only retrieves baseball positions

**Review by ketonkss4 — CHANGES_REQUESTED** [review]

File: lib/util/AbbreviationMappingsHelper.dart (hunk 1 above)
Comment (Friendlygeckos) [reply lib/util/AbbreviationMappingsHelper.dart]: Done

File: lib/util/AbbreviationMappingsHelper.dart (hunk 2 above)
Comment (Friendlygeckos) [reply lib/util/AbbreviationMappingsHelper.dart]: Done

**Review by ketonkss4 — APPROVED** [review]

**Review by pearlson16 — APPROVED** [review]

**Pull request merged**